### PR TITLE
request from Marina Mineeva

### PR DESCRIPTION
**Pull request by mmineeva**
skip pass FakeQuantizeMulFusion

### CODE REVIEW

**Review by Tygrikon — APPROVED** [review]

File: inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp [hunk 1]
```diff
@@ -111,7 +111,8 @@ bool ngraph::pass::ConvertOpSet1ToLegacy::run_on_function(std::shared_ptr<ngraph
     manager.register_pass<ngraph::pass::ConstantFolding>();
 
     // Multiply the thrird and fourth input instead of the output of FQ with all const inputs
-    manager.register_pass<ngraph::pass::FakeQuantizeMulFusion>();
+    // EDIT: skip pass in order to fix accuracy
+    // manager.register_pass<ngraph::pass::FakeQuantizeMulFusion>();
```
Comment (ilyachur) [inline inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp:115]: Does it means that we have a bug in the transformation pass? Or should we disable it for some cases?

Comment (mmineeva) [reply inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp:115]: it was deleted in master branch, but when release branch for kmb plugin was fixed, the pass was still there.

Comment (GlebKazantaev) [reply inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp:115]: I realized that we removed it on master so this PR is just a cherry-pick to kmb branch.

**Review by GlebKazantaev — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Comment by mmineeva** [comment]
build https://openvino-ci.intel.com/job/private-ci/job/github_trigger/job/openvino/job/PR-2953/1
was aborted due to the following issue
"Cannot contact ci-i-ie-e2e-tests-linux-ubuntu18-samples-20830-48f4t-f1pt3: hudson.remoting.ChannelClosedException: Channel "hudson.remoting.Channel@750b2a9e:JNLP4-connect connection from nnlvdp-myriadx-evm08.inn.intel.com/10.125.230.85:51866": Remote call on JNLP4-connect connection from nnlvdp-myriadx-evm08.inn.intel.com/10.125.230.85:51866 failed. The channel is closing down or has closed down"
https://openvino-ci.intel.com/job/private-ci/job/ie/job/e2e-tests-linux-ubuntu18-samples/20830

May I rebuild that somehow? It doesn't look like an issue with my commit.

**Comment by Tygrikon** [comment]
@ababushk Could you please explain if it makes sense to wait for this job? https://openvino-ci.intel.com/job/private-ci/job/github_trigger/job/openvino/job/PR-2953/2/ Or it's a hang

**Comment by ababushk** [comment]
It makes sense to wait, it was waiting for available node and now it is in progress

**Comment by mmineeva** [comment]
@GlebKazantaev build succeeded, please, merge.

**Comment by Tygrikon** [comment]
@ilyachur  ci passed, please merge

**Pull request merged**